lib/pedicel/ec.rb

```diff
@@ -6,10 +6,15 @@ def ephemeral_public_key
       Base64.decode64(@token['header']['ephemeralPublicKey'])
     end
 
+    # Decrypt self.encrypted_data() finding the key from either
+    # symmetric_key, og
+    #
```

Comment:
Rather no comment IMO.

Reply:
I'd like to see comments exactly when I need to read the comments in order to understand the code.

My reasoning:
* If a method needs a comment to understand what the method is doing, then rename the method name and/or parameters.
* If I need to really understand, I'd read the source code anyway.
* Comments will over time drift from the actual code since nothing enforces (or tests) that the comments are aligned with the code. Thus, over time, comments will inevitably be wrong. And, rather no comments than wrong/misleading/... comments.